app/controllers/api/v1/diaper_calculators_controller.rb

```diff
@@ -10,6 +10,8 @@ class DiaperCalculatorsController < ApplicationController
         { name: 'to_be_used_diapers_amount', result: 0 }
       ].freeze
 
+      DIAPER_TITLE = 'diaper'
```

Comment:
https://www.honeybadger.io/blog/when-to-use-freeze-and-frozen-in-ruby/

Reply:
Thanks!

Reply:
Rubocop condemns such a decision